icekit/navigation/templates/icekit/navigation/navigation_item.html

```diff
@@ -0,0 +1,8 @@
+<li class="ik-nav-item {% if instance in active_navigation_items %}ik-nav-item--active{% endif %}">
+    <a
+        class="ik-nav-item-link {% if instance.html_class %}{{ instance.html_class }}{% endif %}"
```

Comment:
I still think the class is best on the outermost tag - what if we want to style the `<li>` for this item differently?